src/sentry/static/sentry/app/views/organizationGroupDetails/eventToolbar.jsx

```diff
@@ -82,6 +86,48 @@ const GroupEventToolbar = createReactClass({
     );
   },
 
+  renderRelatedTransactionButton() {
+    const {organization, event, orgId} = this.props;
+
+    const orgFeatures = new Set(organization.features);
+
+    if (!orgFeatures.has('performance-view')) {
+      return null;
+    }
+
+    const transactionTag = event?.tags?.find(tag => {
+      return tag?.key === 'transaction';
+    });
```

Comment:
You don't need to go into tags to get an error's transaction. You can use `event.transaction`.

Reply:
`event.transaction` didn't work; there was no key `transaction` in the `event` object.

Reply:
Interesting. It is in many of the events I've looked at.